### DISCUSSION TITLE
Create multiple marker groups of different type with `@ create-marker?

### DISCUSSION THREAD

**Discussion by salmankhilji** [kovidgoyal/kitty#5913] [Q&A] [answered]
It appears that I can only create multiple marker groups of the _same_ type with this command:

```
kitty @ create-marker text 1 foo 2 bar 3 baz
```

The syntax does not appear to allow setting group 2 and group 3 to, say, `regex`, and `itext`, like this:

```
kitty @ create-marker text 1 foo regex 2 'bar.*' itext 3 baZ # (bad code)
```

Is this by design?

(Just asking to make sure I am not missing anything.  Not important enough to create a new feature.)

**Comment by kovidgoyal** [comment, marked as answer]
Yes, you can only set *one* marker at a time. The syntax for a marker is

marker-type specification

the specification can contain multiple patterns each assigned to
different groups.
